### PR TITLE
feat(condo): DOMA-3958 new style for popup

### PR DESCRIPTION
**Pull request by Alllex202**
Examples (small & big modals)
Before:
Снимок экрана 2022-09-12 в 16 59 14
After:
Снимок экрана 2022-09-12 в 16 59 30

Before: 
Снимок экрана 2022-09-12 в 17 03 53
After:
Снимок экрана 2022-09-12 в 17 04 01

### CODE REVIEW

File: apps/condo/domains/miniapp/components/IFrameModal.tsx [hunk 1]
```diff
@@ -36,10 +36,9 @@ const IFrameModal: React.FC<IIFrameModalProps> = React.memo((props) => {
             visible
             centered
             footer={null}
-            bodyStyle={MODAL_BODY_STYLES}
-            style={MODAL_STYLES}
+            // bodyStyle={MODAL_BODY_STYLES}
+            // style={MODAL_STYLES}
```
Comment (VKislov) [inline apps/condo/domains/miniapp/components/IFrameModal.tsx:40]: maybe just remove it?

File: apps/condo/domains/subscription/hooks/useEndTrialSubscriptionReminderPopup.tsx [hunk 2]
```diff
@@ -94,7 +94,7 @@ export const useEndTrialSubscriptionReminderPopup = (): IEndTrialSubscriptionRem
             onCancel={handleCloseModal}
             centered
             width={600}
-            bodyStyle={{ padding: '30px' }}
+            // bodyStyle={{ padding: '30px' }}
```
Comment (VKislov) [inline apps/condo/domains/subscription/hooks/useEndTrialSubscriptionReminderPopup.tsx:97]: same

File: apps/condo/domains/subscription/hooks/useServiceSubscriptionWelcomePopup.tsx [hunk 3]
```diff
@@ -85,7 +85,7 @@ export const useServiceSubscriptionWelcomePopup = (): IServiceSubscriptionWelcom
             onCancel={handleCloseModal}
             centered
             width={600}
-            bodyStyle={{ padding: '30px' }}
+            // bodyStyle={{ padding: '30px' }}
```
Comment (VKislov) [inline apps/condo/domains/subscription/hooks/useServiceSubscriptionWelcomePopup.tsx:88]: same

**Comment by SavelevMatthew** [comment]
PR title can be non-semantic 😄 

btw, `style` tag is used for **code-style** changes, but here is a here `feat` / `fix` / `refactor` is more acceptable

**Comment by VKislov** [comment]
Сommit messages must also match the pattern 
`feat(condo): DOMA-XXXX added new analytics`

**Comment by SavelevMatthew** [comment]
> Сommit messages must match the pattern
> `feat(condo): DOMA-XXXX added new analytics`
> but it seems so far the commit messages are not linted

For now listing is disabled in CI to make you merge your old task without a pain. But after some time it will checked here.

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/miniapp/components/IFrameModal.tsx (hunk 1 above)
Comment (Alllex202) [reply apps/condo/domains/miniapp/components/IFrameModal.tsx]: done

File: apps/condo/domains/subscription/hooks/useEndTrialSubscriptionReminderPopup.tsx (hunk 2 above)
Comment (Alllex202) [reply apps/condo/domains/subscription/hooks/useEndTrialSubscriptionReminderPopup.tsx]: done

File: apps/condo/domains/subscription/hooks/useServiceSubscriptionWelcomePopup.tsx (hunk 3 above)
Comment (Alllex202) [reply apps/condo/domains/subscription/hooks/useServiceSubscriptionWelcomePopup.tsx]: done

**Review by dkoviazin — APPROVED** [review]

**Comment by Alllex202** [comment]
Was fixed component Prompt and added separator after header when scrolling modal

**Comment by Alllex202** [comment]
Снимок экрана 2022-09-23 в 21 26 29

**Review by VKislov — APPROVED** [review]

File: apps/condo/domains/common/components/Modal.tsx [hunk 4]
```diff
@@ -1,8 +1,108 @@
...
-    .ant-modal-header {
-      width: calc(100% - 56px);
+import { colors, fontSizes } from '@condo/domains/common/constants/style'
+import { CrossIcon } from '@condo/domains/common/components/icons/CrossIcon'
+
+interface StyledModalProps {
+    isScrolling?: boolean
+}
+
+const StyledModal = styled(AntdModal)<StyledModalProps>`
```
Comment (SavelevMatthew) [inline apps/condo/domains/common/components/Modal.tsx:13]: Would be awesome to move it to `@condo/ui`

Comment (SavelevMatthew) [reply apps/condo/domains/common/components/Modal.tsx:13]: But without CSS-in-JS, pure styling usually working faster :)

**Review by SavelevMatthew — APPROVED** [review]

**Comment by Alllex202** [comment]
waiting https://github.com/open-condo-software/condo/pull/2302

**Comment by Alllex202** [comment]
Adding modal in https://github.com/open-condo-software/condo/pull/2599

**Comment by pahaz** [comment]
Could you please close this pr?

**Pull request closed without merging**